sql/2021/security/tls_ca_issuers_pages.sql

```diff
@@ -0,0 +1,25 @@
+#standardSQL
+#Distribution of CA issuers for all pages
+SELECT
+  client,
+  issuer,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total_pages,
+  COUNT(0) AS freq,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
+FROM (
+  SELECT
+    client,
+    NET.HOST(url) AS request_host,
+    ANY_VALUE(JSON_EXTRACT_SCALAR(payload, '$._securityDetails.issuer')) AS issuer
```

Comment:
We added that as a column this year to make it cheaper/easier to query:

```suggestion
    ANY_VALUE(cert_issuer) AS issuer
```